e2e/internal/kubeclient/deploy.go

```diff
@@ -269,6 +269,9 @@ func (c *Kubeclient) WaitFor(ctx context.Context, condition WaitCondition, resou
 	// watcher doesn't reconnect. To fix this we add another retry loop.
 	retryCounter := 30
 
+	what := fmt.Sprintf("%s %s/%s", resource.kind(), namespace, name)
```

Comment:
Nit: Maybe resourceID/resourceIdentifier or even resourceName is more descriptive? Maybe even ResourceKindNamespaceName in reference to K8s' GroupVersionKind.

Reply:
Opted for `qualifiedResourceName`.